pkg/daemon/update.go

```diff
@@ -52,14 +52,13 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig) error {
 // systemd units. there is no support for multiple filesystems at this point.
 //
 // in addition to files, we also write systemd units to disk. we mask unit files
-// when appropriate, but we ignore requests to enable or disable units.
-// this function relies on the system being restarted after an upgrade,
-// so it doesn't daemon-reload or restart any services.
+// when appropriate. this function relies on the system being restarted after an
+// upgrade, so it doesn't daemon-reload or restart any services.
 //
 // it is worth noting that this function explicitly doesn't rely on the ignition
-// implementation of file and unit writing. this is because ignition is built on
-// the assumption that it is working with a fresh system, where as we are trying
-// to reconcile a system that has already been running.
+// implementation of file, unit writing, enabling or disabling. this is because
+// ignition is built on the assumption that it is working with a fresh system,
+// where as we are trying to reconcile a system that has already been running.
 //
 // in the future, this function should do any additional work to confirm that
 // whatever has been written is picked up by the appropriate daemons, if
```

Comment:
nit: remove "enabling and disabling systemd units should be supported as well" as this changes that (line 66, github won't let me comment on it directly though)

Reply:
Done